documentation/contributing/contributor-guide.adoc

```diff
@@ -1,7 +1,7 @@
 [[contributing-to-docs]]
 = Contributing to the documentation
 
-The following guide walks you through contributing to the Strimzi documentation.
+This procedure shows the steps to contribute to the Strimzi documentation.
 
```

Comment:
_Suggested change:_

This section shows you how to contribute to the Strimzi documentation and contains important guidelines for creating accessible content.